Test.C

```diff
@@ -128,20 +128,22 @@ int main(){
     player.currentCredits = 30;
 
     while(player has options and more rounds exists){
-
-        //output player status (if statements)
-        //current credit
-        //current equipment
-        //pulls from info from index amount table referencing index struct table
-                /* WHY DO WE NEED TO USE MULTIPLE ARRAYS FOR EQUIPMENT INSTEAD OF A LIST WITH A QUANTITY ATTRIBUTE? */
-        //current encrypt
-        //iterate and print through current encryption
+        //receive credits NATHAN
+        if(end of round){ //output player status (if statements)
+            player.currentCredits = player.currentCredits - (equipment.cost + encryption.cost); //current credit
+            player.equipmentInventory[i] = player.equipmentInventory[i]; //current equipment
+            /* using i as the array type for equipment here. whatever the current equipment type is will be put in i */
+            //pulls from info from index amount table referencing index struct table
+            /* WHY DO WE NEED TO USE MULTIPLE ARRAYS FOR EQUIPMENT INSTEAD OF A LIST WITH A QUANTITY ATTRIBUTE? */
+            player.encryptionInventory = {encryptionStock[j]}; //current encrypt
+            /* using j as the array type for encryption here. whatever the current encryption type is will be put in j */
+            //iterate and print through current encryption
+        }
         
-        //receive credits
         //use equation to figure out how many credit user receives
```

Comment:
Add back in comment that I misplaced while fixing merge conflict.
```suggestion
        //receive credits NATHAN
        //use equation to figure out how many credit user receives
```